### ISSUE TITLE
How to disable shortcuts?

### ISSUE THREAD

**Issue by Manius71** [guysoft/FullPageOS#184]
Thank you for your work. There is only one thing I do not know how to solve. I would like to disable the ability to use keyboard shortcuts built into Chromium. How to do it?

**Comment by guysoft** [comment]
Er, I just googled and pasted the first answer from askubuntu, that solves it: https://askubuntu.com/questions/162700/how-can-one-disable-shortcuts-of-chrome

**Comment by Manius71** [comment]
Yes. I used this before you wrote, but this doesn't solve the problem. It deprives me of other important shortcuts: Ctrl+Alt+F(1-7) or Ctrl+Alt+Del.

**Comment by Manius71** [comment]
Let's be serious. Probably everyone starts looking for a solution on Google.

Well... I used this before you wrote, but this doesn't solve the problem. I found the best solution here [Ctrl key lock] (https://stackoverflow.com/questions/5637530/disable-shortcuts-in-google-chrome), but it deprives me of other important shortcuts not related to Chromium: Ctrl+Alt+F(1-7) or Ctrl+Alt+Del.

Thanks for the answer. Although unfortunately not helpful.
